moderator.js

```diff
@@ -0,0 +1,23 @@
+var lang = require('./languages');
+var bindings = require('./bindings');
+var secuence = require('./secuence');
+
+exports.getScriptFromDeck = function (deck) {
```

Comment:
If you're exporting the private function, why not make it public from the beginning?